src/workflows/bidsCopyRawFolder.m

```diff
@@ -2,21 +2,23 @@
 
 function bidsCopyRawFolder(opt, deleteZippedNii, modalitiesToCopy)
```

Comment:
when I run the scripts, I'll see it soon. Is this function still copies EVERY subject in the raw folder or would only copy the corresponding subject that is indicated in `getOption.m` ?

Reply:
only the copies the subject you defined in the `opt` structure.